dace/sdfg/state.py

```diff
@@ -2578,6 +2583,17 @@ def sdfg(self) -> 'SDFG':
 @make_properties
 class AbstractControlFlowRegion(OrderedDiGraph[ControlFlowBlock, 'dace.sdfg.InterstateEdge'], ControlGraphView,
                                 ControlFlowBlock, abc.ABC):
+    """
+    Abstract superclass to represent all kinds of control flow regions in an SDFG.
```

Comment:
docstrings should use double backticks (``) for inline code in Sphinx.

Reply:
Changed